src/components/Card/Card.tsx

```diff
@@ -1,58 +1,74 @@
 import React from "react";
 import "./Card.scss";
 export interface CardProps {
-  style?: React.CSSProperties;
-  cardTitle?: string;
-  className?: string;
-  showBorder?: boolean;
-  children?: React.ReactNode;
-  cardIcon?: string;
-  headerStyle?: React.CSSProperties;
+  title?: string;
+  image?: JSX.Element;
+  imageFocus?: "top" | "bottom" | "left" | "right" | "center";
+  imageRatio?: "wide" | "landscape" | "portrait" | "square";
+  orientation?: "horizontal" | "vertical";
+  description?: string | Element;
+  primaryCTA?: JSX.Element;
+  secondaryCTA?: JSX.Element;
+  caption?: JSX.Element;
+  header?: JSX.Element;
+  footer?: JSX.Element;
 }
 
 const Card = (props: CardProps) => {
   const {
-    style,
-    headerStyle,
-    cardTitle,
-    className,
-    showBorder,
-    children,
-    cardIcon,
+    title,
+    image,
+    imageFocus,
+    imageRatio,
+    orientation,
+    description,
+    primaryCTA,
+    secondaryCTA,
+    caption,
+    footer,
+    header,
     ...restProps
   } = props;
   return (
     <div
-      className={
-        className
-          ? `${className} ${
-              showBorder && "nitrozen-card-container-border"
-            } nitrozen-card-container`
-          : `${
-              showBorder && "nitrozen-card-container-border"
-            } nitrozen-card-container`
-      }
-      style={style}
-      {...restProps}
+      className={`n-card n-card__container ${
+        orientation === "vertical" && "card-vertical"
```

Comment:
We should be avoiding putting condition in the template, we can make a variable
`
const containerClassname = orientation === "vertical" && "card-vertical";

.....
className={`n-card n-card__container ${containerClassname}`}
`
Just an suggestion.

In case of more complicated prefer moving it to customHook,

`
const useCustomClass = () => {
    // your code
 }
`